### PR TITLE
Issue 41 - main.defaultGpkg() test coverage - take 2!

### PR DESCRIPTION
**Pull request by jlucktay**
From #41 and based on a non-malfunctioning branch this time. 👍

### CODE REVIEW

File: util.go [hunk 1]
```diff
@@ -0,0 +1,71 @@
+///////////////////////////////////////////////////////////////////////////////
+//
+// The MIT License (MIT)
+// Copyright (c) 2018 Jivan Amara
+// Copyright (c) 2018 Tom Kralidis
+//
```
Comment (JivanAmara) [inline util.go]: Add your John Hancock here.

File: util_test.go [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+			testSubPath:   "3",
+			dummyFileName: path.Join(baseTestPath, "3/test_data/testDataDummy.gpkg"),
+		},
+		{
+			testSubPath:   "4",
+			dummyFileName: "",
+		},
+	}
+
+	for _, c := range cases {
```
Comment (JivanAmara) [inline util_test.go]: Collect the index here...

File: util_test.go [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+		casePath := path.Join(baseTestPath, c.testSubPath)
+
+		if chdirErr := os.Chdir(casePath); chdirErr != nil {
+			t.Errorf("Could not change working directory to '%s'.", casePath)
```
Comment (JivanAmara) [inline util_test.go]: ...and prefix error messages w/ [<case index>]

File: util_test.go [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+		}
+
+		got := defaultGpkg()
+
+		if got != c.dummyFileName {
+			t.Errorf("Under '%s', defaultGpkg() == '%s', wanted '%s'.", casePath, got, c.dummyFileName)
```
Comment (JivanAmara) [inline util_test.go]: ...and prefix error messages w/ [<case index>]

**Review by JivanAmara — CHANGES_REQUESTED** [review]
Looks good.  A few minor changes.  Don't forget to add your name to the copyright section of files you modify & remove other devs names from files that they didn't contribute to ( yet ;-) ).

**Comment by jlucktay** [comment]
OK, all done @JivanAmara 👍

**Comment by JivanAmara** [comment]
@jlucktay All tests passing?

**Comment by JivanAmara** [comment]
@jlucktay ping, tests passing?

**Comment by jlucktay** [comment]
They were passing when I last committed, but now there is a problem in the `data_provider` package with `geom.Extent`.

**Comment by jlucktay** [comment]
All sorted. Passing on MacOS and Windows both!

**Comment by JivanAmara** [comment]
Great, will merge.  Appreciate your help, if you've got more time & interest I'd love to have you take on another piece :-)

**Pull request merged**